### PR TITLE
admin ui

### PR DESCRIPTION
**Pull request by GaganCJ**
just the appbar and little changes in style properties

### CODE REVIEW

**Comment by GaganCJ** [comment]
added driver ui

File: views/admin/adminui.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <link rel="shortcut icon" href="../img/Graphicloads-Transport-Bus-4.ico" type="image/x-icon">
+    <link rel="icon" href="../img/Graphicloads-Transport-Bus-4.ico" type="image/x-icon">
+
+    <link href="https://cdn.metroui.org.ua/v4/css/metro-all.min.css" rel="stylesheet">
+    <link href="../css/mainui.css" rel="stylesheet">
```
Comment (Gunashree-R-V) [inline views/admin/adminui.html]: /css/mainui.css instead of ../css/mainui.css

File: views/admin/adminui.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    <title>Administration Dashboard</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="app-bar cappbar bg-teal z-dropdown" data-role="appbar">
+            <h3 class="brand no-hover">
+                <span style="width: auto; line-height: 35px;" class="text-center border-none bd-darkGray pl-2 pr-2 mt-2 fg-white"><span
+                        class="pr-4"><img src="../img/Graphicloads-Transport-Bus-4.ico" height="40px;"></span>Transport
```
Comment (Gunashree-R-V) [inline views/admin/adminui.html]: Same here also. /img intstead of ../img

File: views/driver/driverui.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                                        <li data-icon="<span class='mif-map'>" data-caption="Route-No. : "></li>
+                                        <li data-icon="<span class='mif-bus'>" data-caption="Vehicle-No. : "></li>
+                                        <li data-icon="<span class='mif-users'>" data-caption="Students Count : "></li>
+                                        <li data-icon="<span class='mif-alarm'>" data-caption="Time : "></li>
+                                    </ul>
+                                    <button class="button bg-green rounded drop-shadow">Track Location</button>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="frame">
```
Comment (Gunashree-R-V) [inline views/driver/driverui.html]: Can we have only one trip instead of one week? Sorry for the inconvenience.

File: views/driver/driverui.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                        data-active-content-class="bg-dark fg-white">
+                        <div class="frame active">
+                            <div class="heading text-center">22 Oct</div>
+                            <div class="content">
+                                <div class="p-2">
+                                    <ul data-role="listview" class="mb-2 rounded newnode">
+                                        <li data-icon="<span class='mif-map'>" data-caption="Route-No. : "></li>
+                                        <li data-icon="<span class='mif-bus'>" data-caption="Vehicle-No. : "></li>
+                                        <li data-icon="<span class='mif-users'>" data-caption="Students Count : "></li>
+                                        <li data-icon="<span class='mif-alarm'>" data-caption="Time : "></li>
```
Comment (Gunashree-R-V) [inline views/driver/driverui.html]: Date isn't added?Is it not required?

File: views/driver/driverui.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+                    class="mif-arrow-left mif-2x"></span></a>
+                <div class="avatar">
+                    <img data-role="gravatar" src="../img/user.png" data-role-gravatar="true">
+                </div>
+                <span class="title csidebar-title fg-white">Name</span>
+                <span class="subtitle csidebar-subtitle fg-white">Driver-ID</span>
+            </div>
+            <ul class="sidebar-menu">
+                <li><a><span class="mif-home fg-red icon"></span>Home</a></li>
+                <li><a><span class="mif-books icon"></span>Guide</a></li>
```
Comment (Gunashree-R-V) [inline views/driver/driverui.html]: Can the unnecessary icons like Guide be removed?

Comment (GaganCJ) [reply views/driver/driverui.html]: it is just for reference, yes it can be replaced with other icons w.r.t their features

**Pull request merged**